aries-site/src/pages/templates/forms.mdx

```diff
@@ -250,3 +252,41 @@ In cases where the user is given an opportunity to express themselves or would l
   href="#sign-in"
   alt="HPE Design System Form where required and optional can be skipped"
 />
+
+## Full-page form guidance
+
+- Use full-page forms for create and edit flows, irrespective of the size of the form.
+- Use a single-column full-page form instead of multi-column layouts.
+- Full-page forms shoud be center-aligned.
+- Use full-page when the form is too long.
+- Use Center modal instead of side drawers/layers for secondary inputs.
+- Use side layers to display help text, filters, or a form over another form.
+
+### Full Page Form Example
+
+<CardGrid cards={formLayouts} />
+
+### Avoid using multiple column layouts
+
+Single-column layouts create clear paths for users.
+Multi-column layouts can inadvertently cause users to skip important fields, input data in the wrong field, or simply prevent progress; all of which can lead users to abandon forms.
+
+<BestPracticeGroup>
+  <Example
+    bestPractice={{
+      type: 'do',
+      message:
+        'One column layer to create a clear path to completion for the user.',
+    }}
+  >
+    <ShippingExample />
```

Comment:
Seems like the "do" should be the full-page form example and the "don't "should be an altered version of that example that's 2 columns so that it's clear what the difference between the do and don't are

Reply:
okay I can change that to look like full page form